main.go

```diff
@@ -100,6 +102,14 @@ func main() {
 		panic(fmt.Errorf("can't use both schema-only and skip-foreign-keys at once. Foreign Key creation can only be skipped when data migration takes place."))
 	}
 
+	if targetDb == conversion.TARGET_SPANGRES {
+		if !(driverName == conversion.PGDUMP || driverName == conversion.POSTGRES) {
+			panic(fmt.Errorf("cannot convert to spangres when source is not postgres target: %s driver: %s", targetDb, driverName))
+		}
+	} else if targetDb != conversion.TARGET_SPANNER {
+		panic(fmt.Errorf("unkown taget %s", targetDb))
```

Comment:
nit: typo in target

Reply:
How about being a bit more explicit (and tie to specific flag name):

panic(fmt.Errorf("unknown target-db: %s", targetDb))

Reply:
done